packages/components/.gitignore

```diff
@@ -30,3 +30,6 @@ coverage
 
 test-results/
 playwright-report/
+
+# Storybook
+storybook-static/
```

Comment:
CI seems to be expecting this to deploy

Reply:
How do you mean?

Reply:
this PR and main is failing

Reply:
It's looked like the main build was failing yesterday due to and install issue, I'll have a look!